pkg/controllers/status/cluster_status_controller.go

```diff
@@ -244,7 +244,7 @@ func (c *ClusterStatusController) buildInformerForCluster(cluster *v1alpha1.Clus
 	return singleClusterInformerManager, nil
 }
 
-func (c *ClusterStatusController) initLeaseController(cluster *v1alpha1.Cluster) {
+func (c *ClusterStatusController) initLeaseController(cluster *v1alpha1.Cluster, stopCh <-chan struct{}) {
```

Comment:
Would it be better to take `context` as a parameter than a `chan`?

Reply:
Good idea.